apps/website/blog/2024-12-01-maci-getting-started.md

```diff
@@ -0,0 +1,294 @@
+
+```bash
+pnpm merge:[network] --poll [poll-id]
+```
+
+Then you need to generate the proofs with the following command:
+
+```bash
+pnpm run prove:[network] --poll [poll-id] \
+    --coordinator-private-key [coordinator-maci-private-key] \
+    --tally-file ../proofs/tally.json \
+    --output-dir ../proofs/proofs/ \
+    --start-block [block-number]  \
+    --blocks-per-batch [number-of-blocks]
+```
+
+:::important
+You can reduce the time of the proving by including more blocks per batch, you can try with 500.
+:::
+
+#### Submit On-chain
+
+Now it's time to submit the poll results on-chai so that everyone can verify the results:
+
+```bash
+pnpm submitOnChain:[network] --poll [poll-id] \
+    --output-dir proofs/ \
+    --tally-file proofs/tally.json
+```
+
+#### Tally
+
+Once the proofs are generated, and results tallied, the results (Tally) are written to a file. Let's take a look at one:
+
+```json
+{
+  "maci": "0xd54b47F8e6A1b97F3A84f63c867286272b273b7C",
+  "pollId": "0",
+  "network": "localhost",
+  "chainId": "31337",
+  "isQuadratic": true,
+  "tallyAddress": "0xD4fbAF1dFe100d07f8Ef73d8c92e93d0Bcf7b45D",
+  "newTallyCommitment": "0x2f55cc85f7f141098ba791a9f6a646f8773b9bb4f5852ccc33b5a28e7b0756e5",
+  "results": {
+    "tally": [
+      "9",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0"
+    ],
+    "salt": "0x2e9cd240b86cf456fa4deced8e7420c45e3c16941d2dcec308f8b6d48264dda3",
+    "commitment": "0x296eac2a7289974f23497bebd39e86599d0b7032796fb84dcc1f6bbda38262ca"
+  },
+  "totalSpentVoiceCredits": {
+    "spent": "81",
+    "salt": "0x24f57b75c227987727c13d1e83409d70478b42bdc12a4a4df8129c72fbaf5aaf",
+    "commitment": "0xb4ebe68b0da828c0b978ddee86ba934b8e215499ac766491f236ad85fd606de"
+  },
+  "perVOSpentVoiceCredits": {
+    "tally": [
+      "81",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0",
+      "0"
+    ],
+    "salt": "0x2590434ea2d600f7bd2396ba7fa454ad4c975c29424ee481561d9786538a5e48",
+    "commitment": "0x54ec996599886da21c4b07c25d1de544292a8b7c38b79726995c869c9e95db"
+  }
+}
+```
+
+We see that there is an array named results, this contains the aggregated votes for each option, where each option is represented by an index in the array. In this case above, the first option (index 0) received a total of 9 votes, where every other option did not receive any votes.
```

Comment:
suggest by chatgpt: "We observe an array named `results`, which holds the aggregated votes for each option. Each option corresponds to an index in the array. In the example above, the first option (index 0) received a total of 9 votes, while all other options received no votes"